github_pr.py

```diff
@@ -436,7 +439,8 @@ def main():
     parser.add_argument('--mergecomment', default=":shipit:", help='string to look for when checking comments for "shipit" approval, during MERGE only')
     parser.add_argument('--condition-non-owner-merger', action='store_true', help='stops owner from being able to apply merge comment')
     parser.add_argument('--condition-approved-mergers', default=None, nargs='+', help='list of usernames of approved mergers')
-    parser.add_argument('--condition-approved-mergers-file', default='./MAINTAINERS.txt', nargs='?', help='file of usernames of approved mergers')
```

Comment:
when we looked at doing some other stuff with our jenkins job, it simplified it a fair bit to not have to specify the file (we were specifying it in bash, then setting it to empty multiple times in a loop)